definitions/npm/parse_v1.9.x/flow_v0.30.x-/parse_v1.9.x.js

```diff
@@ -201,8 +201,8 @@ declare type $npm$parse$ByIdMap = { [userId: string]: $npm$parse$PermissionsMap
 
 declare class $npm$parse$ParseACL {
   permissionsById: $npm$parse$ByIdMap,
-  static(arg1: $npm$parse$ParseUser | $npm$parse$ByIdMap): $npm$parse$ParseACL,
-  constructor(arg1: $npm$parse$ParseUser | $npm$parse$ByIdMap): $npm$parse$ParseACL,
+  static(arg1: $npm$parse$ParseUser | $npm$parse$ByIdMap | void): $npm$parse$ParseACL,
+  constructor(arg1: $npm$parse$ParseUser | $npm$parse$ByIdMap | void): $npm$parse$ParseACL,
```

Comment:
Would `constructor(arg1?: $npm$parse$ParseUser | $npm$parse$ByIdMap)` be clearer?

Reply:
Definitely!